src/fields/Fields.cpp

```diff
@@ -67,8 +67,8 @@ Fields::AllocData (
 
         if (m_explicit) {
             // explicit solver:
-            // beams share rho_beam jx_beam jy_beam jz_beam
-            // rho jx jy jz for all plasmas and beams
+            // beams share jx_beam jy_beam jz_beam
+            // jx jy rhomjz for all plasmas and beams
```

Comment:
```suggestion
            // jx jy rho rhomjz for all plasmas and beams
```
Since rho still exists (although just for diagnostics) I think we should keep it here